### ISSUE TITLE
dataDirectory = null will silently be ignored

### ISSUE THREAD

**Issue by frk1** [DoctorMcKay/node-steam-user#74]
Please excuse me if I'm dumb, but I experienced undefined behaviour while trying to disable the dataDirectory option.

The readme says that
> Controls where the Steam server list and sentry files are written. If null, no data will be automatically stored.

But when I look at the code, setting it to `null` will set the default directories anyways, resulting in data being written.

[index.js](https://github.com/DoctorMcKay/node-steam-user/blob/d832ede75b959a39874a87985fdc8534a837b6c1/index.js#L89-L97)
```js
if (!this.options.dataDirectory) {
  if (process.env.OPENSHIFT_DATA_DIR) {
    this.options.dataDirectory = process.env.OPENSHIFT_DATA_DIR + "/node-steamuser";
  } else {
    this.options.dataDirectory = (new AppDirectory({"appName": "node-steamuser", "appAuthor": "doctormckay"})).userData();
  }
}

this.storage = new FileStorage(this.options.dataDirectory);
```

Is this a bug or am I missing a difference between calling the `!` operator on an unset property vs calling it on a property set to `null`? 

_In case you are curious what the undefined behaviour was: I was trying to save the sentry manually. I would receive a sentry every time, but it was never confirmed to the Valve servers. The reason for that was an existing (confirmed) sentry file (in the default dataDirectory) from previous experiments with the same account. When trying to login at a later point the logOn method would always try to use the new and unconfirmed sentry file I saved earlier, thus resulting in a failure._

Greetings

_Btw, the git repository link for your `file-manager` package is dead, you might consider updating it on npm._

**Comment by frk1** [comment]
I thought about opening a PR first, but I don't know if you prefer to keep the "undefined behaviour" in place (for backwards compatibility) and how your preferred solution for "really" disabling the dataDirectory would look like so I decided against it.

**Comment by DoctorMcKay** [comment]
This should be fixed in v3.11.0.
